staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/metrics/metrics.go

```diff
@@ -25,10 +29,13 @@ import (
 )
 
 const (
-	namespace        = "apiserver"
-	subsystem        = "envelope_encryption"
-	FromStorageLabel = "from_storage"
-	ToStorageLabel   = "to_storage"
+	namespace                 = "apiserver"
+	subsystem                 = "envelope_encryption"
```

Comment:
couldn't we also add these metrics to the other encryption providers and not just kms? Or at least the new KEK-related ones since all the other provider have KEKs